doc/api/tty.md

```diff
@@ -121,6 +121,29 @@ added: v0.7.7
 A `number` specifying the number of rows the TTY currently has. This property
 is updated whenever the `'resize'` event is emitted.
 
+### writeStream.getColorDepth([env])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `env` {object} A object containing the environment variables to check.
+  Defaults to `process.env`.
+* Returns: {number}
+
+Returns:
+* 1 for 2,
+* 4 for 16,
+* 8 for 256,
+* 24 for 16,777,216
+colors supported.
+
+Use this to determine what colors the terminal supports.
+Due to the nature of colors in terminals it is possible to either have false
+positives or false negatives.
```

Comment:
It might be worth noting in the docs that this determination is based on a set of inspection heuristics that may not be 100% accurate in every case. It really is dependent on the environment and the environment could, in fact, lie.

Reply:
Done